evasquez26/angular4-startup
# S3-ONE-129 Request roles: Admin, Super Admin, Moderator

## Story / Task

see [ONE-129](https://bixlabs.atlassian.net/browse/ONE-129)


## What this PR does?

Adds functionality to request roles: admin, super admin, moderator.
(The functionality to request roles was basically introduced before, but this PR adds fixes, and improvements)

### How to test / reproduce

- Login to the app
- If the user has admin privileges you should see an Admin menu (new)
- Go to My Profile -> Roles
- You should be able to request any role
- You should be able to request a role under an organization or in the global scope
- You should be able to see the request in the user's roles list as pending


### TODOs
- [x] Rebase after merging #33 

### Related PRs
N/A

## Migrations
 NO


Thank you
